### PR TITLE
Fixed issue99

### PR DESCRIPTION
**Pull request by gottumuk**
*Issue # 99, if available:*

*Description of changes:*
Modified the delete_global_cluster function to check for promotion of all secondary clusters to standalone clusters and delete them after promotion

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: global-clusters-automation/failover_and_delete_global_cluster.py [hunk 1]
```diff
@@ -148,9 +148,9 @@ def delete_global_cluster(global_cluster_id, secondary_cluster_arn, secondary_cl
         if each_secondary_cluster != secondary_cluster_arn:
             print('Removing secondary cluster ', each_secondary_cluster, ' from global cluster ', global_cluster_id)
             remove_from_global_cluster(each_secondary_cluster, global_cluster_id)
-    # Wait until all standalone clusters are promoted
-    print('Waiting till all secondary clusters are removed from global cluster ', global_cluster_id)
-    wait_for_promotion_to_complete(global_cluster_id, secondary_cluster_arn)
+             # Wait until all standalone clusters are promoted
```
Comment (tmcallaghan) [inline global-clusters-automation/failover_and_delete_global_cluster.py:151]: Remove one space, indedented 1 space too far.

File: global-clusters-automation/failover_and_delete_global_cluster.py [hunk 2]
```diff
@@ -148,9 +148,9 @@ def delete_global_cluster(global_cluster_id, secondary_cluster_arn, secondary_cl
...
+            print('Waiting till all secondary clusters are removed from global cluster ', global_cluster_id)
```
Comment (tmcallaghan) [inline global-clusters-automation/failover_and_delete_global_cluster.py:152]: Modify message to be more relevant, code is actually waiting for the secondary promotion isn't it?

**Review by tmcallaghan — CHANGES_REQUESTED** [review]
Please change as per comments.

File: global-clusters-automation/failover_and_delete_global_cluster.py (hunk 2 above)
Comment (gottumuk) [reply global-clusters-automation/failover_and_delete_global_cluster.py:152]: Code is actually checking for promotion of all secondary clusters to stand alone clusters

**Review by tmcallaghan — APPROVED** [review]
Approved.

**Pull request merged**